### DISCUSSION TITLE
In the long run, the user guide should be re-written in English using LaTeX and titled in pure English.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [goodluck1982/SpaceGroupIrep#21] [General]
I noticed that [the user guide](https://github.com/goodluck1982/SpaceGroupIrep/blob/main/doc/SpaceGroupIrep%E7%A8%8B%E5%BA%8F%E5%8C%85%E8%AF%B4%E6%98%8E.docx) is currently written in Chinese. But I think, in the long run, the document should be re-written in English using LaTeX, and titled in pure English to facilitate the communication between international peers.
 
BTW, the instant communication method, like [WeChat group](https://github.com/goodluck1982/SpaceGroupIrep#communications), is not suitable for in-depth and systematic academic communication. Most people and students just want guidance, rather than trying to find out the details they should know to better improve themselves and help others. This is also why I find it very difficult to establish a real instant messaging academic exchange group. Only teams or communities of people who want to do serious research and who are willing to work hard and have fun doing it are really meaningful scientific research groups.

Regards,
HZ

**Comment by goodluck1982** [comment]
Thanks for your suggestion. 
In fact, the document is not a user guide in the real sense. It's only a Chinese introduction or a lite-version translation of the SpaceGroupIrep paper. The SpaceGroupIrep paper is the best introduction to the package. And the "examples/document and examples.nb" file (in English) is another reference.
